sdmetrics/statistical/univariate/base.py

```diff
@@ -29,19 +29,31 @@ def metric(real_column, fake_column):
         raise NotImplementedError()
 
     def metrics(self, metadata, real_tables, synthetic_tables):
-        tables = set(real_tables).union(synthetic_tables)
-        for name in tables:
-            dtypes = metadata.get_dtypes(name)
-            real = real_tables[name]
-            fake = synthetic_tables[name]
-            yield from self._handle(name, dtypes, real, fake)
-
-    def _handle(self, name, dtypes, real, fake):
+        """
+        This function iterates over all the columns in all the tables and, if the
+        data types match, it yields the Metric.
```

Comment:
Perhaps change to "This function iterates over all the columns in all the tables and, if the data type of a column matches the data type for which this metric is defined, it computes the metric for that column and yields it."

Reply:
Done.